src/main/java/com/alphawallet/attestation/demo/Demo.java

```diff
@@ -227,10 +227,11 @@ private static void receiveCheque(Path pathUserKey, Path chequeSecretDir,
     }
     // TODO how should this actually be?
     SmartContract sc = new SmartContract();
+    /*FullProofOfExponent newPok = new FullProofOfExponent(redeem.getDerEncoding());
```

Comment:
This should actually be UsageProofOfExponent. If that is used then things seem work as is.

Reply:
Ok